exercises/linked-list/test/test_linked_list.c

```diff
@@ -1,115 +1,124 @@
-#include <stddef.h>
+#include "../src/linked_list.h"
 #include <stdbool.h>
```

Comment:
I may have missed something, but it seems like `stdbool` is not used.

Reply:
Removed.